applications/protocols_serialization/server/snippets/debug/debug.conf

```diff
@@ -15,15 +15,16 @@ CONFIG_LOG=y
 
 # Use separate thread for logging
 CONFIG_LOG_MODE_DEFERRED=y
+CONFIG_LOG_BUFFER_SIZE=4096
```

Comment:
1. Is it possible to pass 4k logging buffer over the serialization interface?
2. If retained memory area is 1k why logging buffer is 4k?